hack/build/run-linters.sh

```diff
@@ -4,4 +4,4 @@ GOLANGCI_VERSION="${GOLANGCI_VERSION:-v1.52.2}"
 
 go install "github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_VERSION}"
 
-golangci-lint run --timeout=8m ./...
+golangci-lint run --timeout=16m ./...
```

Comment:
I did exactly the same thing in #2709

Reply:
I know, just wanted to make the cdi-lint lane pass until your PR will get merged

Reply:
I meant this in a "great minds think alike" kind of way